src/draive/evaluators/truthfulness.py

```diff
@@ -0,0 +1,87 @@
+from draive.evaluation import EvaluationScore, evaluator
+from draive.generation import generate_text
+from draive.types import Multimodal, MultimodalTemplate, xml_tag
+
+__all__ = [
+    "truthfulness_evaluator",
+]
+
+
+INSTRUCTION: str = """\
+Assistant is an evaluator scoring the provided content.
+
+<INSTRUCTION>
+Compare the REFERENCE and the EVALUATED content by carefully examining them, then rate \
+the EVALUATED content using solely a truthfulness (factual accuracy) metric according to the EVALUATION_CRITERIA.
+Think step by step and provide explanation of the score before the final score.
+Use the explained RATING scale and the requested FORMAT to provide the result.
+</INSTRUCTION>
+
+<EVALUATION_CRITERIA>
+Evaluated metric is truthfulness (factual accuracy) - the extent to which the EVALUATED content is factually correct and \
```

Comment:
Evaluated metric is truthfulness (factual accuracy) - This metric evaluates the factual correctness of the content, regardless of its relation to the source material. Truthfulness is concerned with whether the information presented is accurate in the broader context, ensuring that the facts are correct and reliable, and that the content does not perpetuate errors or falsehoods. This metric is more concerned with the overall factual integrity of the content, meaning that even if a claim is not directly traceable to the provided source, it must still be correct and represent the truth of the matter. Truthfulness is about the correctness of the facts themselves, not just their alignment with a specific source.